src/pages/test-report/cat-d/components/vehicle-checks/vehicle-checks.cat-d.ts

```diff
@@ -17,7 +17,7 @@ interface ComponentState {
 }
 
 @Component({
-  selector: 'vehicle-checks-cat-d',
+  selector: 'vehicle-checks-test-report-cat-d',
```

Comment:
Can we change the file and the class name as well?

Reply:
Good point. I'll change that now.